skycultures/tikuna/description.en.utf8

```diff
@@ -0,0 +1,39 @@
+<p align="justify">Priscila Faulhaber (Museum of Astronomy and Related Sciences)</p>
+ <p align="justify">Fernando Vieira (Astronomer and Former Coordinator of Astronomy in The Planetarium Foundation of the City of Rio de Janeiro) helped with the interpretation of the astronomical correlations between Western and Tikuna constellations, identified by the Tikuna in November 2002, during a research visit at the Science Center and Planetarium of the Universidade do Pará (Belém/Brazil), using the Starry Night System. Vieira plotted the correspondence between the traces of Tikuna mythical animals and the Official IAU constellations of western astronomy. He also reviewed this text for any skyscape incongruences related to corresponding Western astronomy and Tikuna cosmovision.</p>
+<p align="justify"We are integrating in this investigation a Physics Professor from the Pontifícia Universidade Católica de São Paulo (PUC-SP) and researcher of astronomy in culture, Walmir Thomazi Cardoso; who made some reinterpretations in a collaborative work with physics undergraduate student and, Youssif Ghantous Filho, adapting and anchoring original drawings to the Stellarium. Nevertheless, every sky interpretation is contextual and circumstantial. Current collaborative investigations are looking for new astronomical observation together with the Tikuna, which will improve the initial findings.</p>
```

Comment:
typos:
- a physics professor (no capital letters)

Reply:
ok I'm already making the changes